spyder/plugins/editor/widgets/base.py

```diff
@@ -1318,4 +1000,6 @@ def wheelEvent(self, event):
                         self.zoom_in.emit()
                 return
         QPlainTextEdit.wheelEvent(self, event)
+        self.completion_widget.hide()
+        QToolTip.hideText()

```

Comment:
Maybe this could be factored out to a helper method?

```python
def hide_completion_widget(self):
        self.completion_widget.hide()
        QToolTip.hideText()
```